### PR TITLE
feat: update-ui, charts page

### PR DESCRIPTION
**Pull request by qiweiii**
Note:
- y-axis ticks cannot be further customized 
image

- brush cannot be further customized 
image

### CODE REVIEW

**Comment by Kirl70** [comment]
Can the style of the brush be similar to that of the CKB browser, the current style user may not know that it can interact

image

**Comment by qiweiii** [comment]
@Kirl70 Seems cannot do the same as ckb explorer as we use different chart package here, but I just found a way to customize those style, let me try and see how it goes

**Comment by qiweiii** [comment]
@FrederLu @Keith-CY Ready for review

**Comment by Kirl70** [comment]
Can the distance between the date and the table be further? It is 16px on the design draft @qiweiii 

image

File: components/ChartComponents/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  if (active && payload && payload.length) {
+    return (
+      <div
+        className={styles.content}
+        style={{
+          border: 'none',
+          borderRadius: isMobile ? 4 : 8,
+          boxShadow: isMobile ? '0px 3px 10px rgba(0, 0, 0, 0.1)' : '0px 6px 20px rgba(0, 0, 0, 0.1)',
+          padding: isMobile ? '12px 8px 10px' : '16px 16px 14px',
+          backgroundColor: '#fff',
```
Comment (Keith-CY) [inline components/ChartComponents/index.tsx:14]: why not include theme style config in `styles.module.scss`

**Comment by qiweiii** [comment]
updated https://godwoken-explorer-53iqocibx-nervosnet.vercel.app/

**Comment by Keith-CY** [comment]
image
Symbol has weird background that covers each other

**Comment by Keith-CY** [comment]
> image Symbol has weird background that covers each other

After checking the design, I found it's as expected

**Review by Keith-CY — APPROVED** [review]

**Comment by Kirl70** [comment]
The color dots in the table should always be on the top layer, 1 does not seem to be on the top layer, and 2 is correct

1、image

2、image

**Comment by FrederLu** [comment]
image
image
@Kirl70  Now the brush is the expected style, I think it is different from the ui.

**Comment by qiweiii** [comment]
@FrederLu We are using `rechart` library for charts, and unfrotunately the border and behaviour of brush cannot be further customized

**Comment by FrederLu** [comment]
image

image

@qiweiii 
The following problem is displayed on the mobile phone：
1、The page can be swiped to the left, and a blank area will appear on the right side of the page.
2、The Average Gas Limit scale information is missing to the right of the icon.
3、When the specific information in the icon is too long, the display is incomplete.

**Comment by qiweiii** [comment]
@FrederLu 

1. the problem is caused by another previous PR, not related to my code in this PR, and it will be fixed in another separate PR later on, please ignore this problem in this PR
2. the design is not showing the y-label on the right in mobile screen size, so I did not show it on mobile (I guess it's because mobile screen is small and showing both y-label will be make diagram too narrow)
3. it won't happen after 1 is fixed

**Comment by qiweiii** [comment]
@Kirl70 After some experiments, the active dot on the line cannot be made on top, it's because the chart library we are using always draw the green line first, then green dot, then the blue line, then blue dot. The first element is painted first, and seems I cannot change that sequence and it's layer

image

Maybe use a smaller dot? or we may stick to the current result

**Comment by Kirl70** [comment]
> @Kirl70 After some experiments, the active dot on the line cannot be made on top, it's because the chart library we are using always draw the green line first, then green dot, then the blue line, then blue dot. The first element is painted first, and seems I cannot change that sequence and it's layer
> 
> image
> 
> Maybe use a smaller dot? or we may stick to the current result

The current effect is acceptable, the impact is not great

**Comment by FrederLu** [comment]
> @FrederLu
> 
> 1. the problem is caused by another previous PR, not related to my code in this PR, and it will be fixed in another separate PR later on, please ignore this problem in this PR
> 2. the design is not showing the y-label on the right in mobile screen size, so I did not show it on mobile (I guess it's because mobile screen is small and showing both y-label will be make diagram too narrow)
> 3. it won't happen after 1 is fixed

@Kirl70 
Does the second point here need to be displayed on the phone?

**Comment by Kirl70** [comment]
There are not many cases described on the left and right sides, I think it can be added 
@qiweiii 
image

**Comment by FrederLu** [comment]
> There are not many cases described on the left and right sides, I think it can be added @qiweiii https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=2276%3A14976 image

@Kirl70 
Whether the Average Gas Limit y-axis has no scale will mistakenly use the same scale as Average Gas Used to give the user the false impression that the data is wrong.

**Comment by Kirl70** [comment]
Regarding the scale, I have two ideas.
1. Hide the information on the left and right sides, and users can click and view the information
image

2. List the scale information below the table
image


Which one do you think is better, or do you have a better idea? @qiweiii @FrederLu

**Comment by FrederLu** [comment]
Compared with Idea 1, I think it will be easier for users to understand the meaning of the diagram after reading Idea 2.

**Comment by Kirl70** [comment]
So please make adjustments according to the second idea, and only make adjustments in the case where there are 2 pieces of information on the mobile terminal @qiweiii 
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=2289%3A16097

**Comment by qiweiii** [comment]
Update: https://godwoken-explorer-dhe0k17aj-nervosnet.vercel.app/

image

note that the legend is below the x-axis, cannot change that part

**Comment by Kirl70** [comment]
Dates in the middle should center the chart

image

**Comment by qiweiii** [comment]
> Dates in the middle should center the chart
> 
> image

Cannot be fixed as those positions are calculated internally by the library

**Review by FrederLu — APPROVED** [review]

**Pull request merged**